pop-cli-for-appchains/pop-cli/call.md

```diff
@@ -0,0 +1,74 @@
+---
+description: Call a chain
```

Comment:
Lets not make this file, see https://github.com/r0gue-io/pop-docs/issues/14

Reply:
I think we should stick with the existing style for consistency, at least until the refactor is done.

Reply:
Please make my changes I suggested in the other file here then as well :)